### PR TITLE
Session2ex1

### PR DESCRIPTION
**Pull request by cubann**

### CODE REVIEW

**Comment by cubann** [comment]
Hope I did not mess with the flow:)

File: home-assignments/session2/cli.py [hunk 1]
```diff
@@ -4,26 +4,34 @@
...
-    temp_high = forecasts[-1].high
-    temp_condition = city_weather_info.condition.text
-
-    print(f'The weather in {city} today is {temp_condition} with temperatures trailing from {temp_low}-{temp_high} {scale_name}.')
+        if scale == 'c':
+            unit = Unit.CELSIUS
+            scale_name = 'celcius'
+        elif scale == 'f':
+            unit = Unit.FAHRENHEIT
+            scale_name = 'fahrenheit'
```
Comment (denzalman) [inline home-assignments/session2/cli.py]: here you don't need this if-else... api understand "f" and "c" letters. simply  Unit.CELSIUS == ''c", and Unit.FAHRENHEIT == "f"

File: home-assignments/session2/cli.py [hunk 2]
```diff
@@ -4,26 +4,34 @@
...
-        scale_name = 'celcius'
-    elif scale == 'f':
-        unit = Unit.FAHRENHEIT
-        scale_name = 'fahrenheit'
+def get_forecast(city, forecast, scale):
 
-  
-    city_weather_info = Weather(unit).lookup_by_location(city)
-    forecasts = city_weather_info.forecast
+    try:
```
Comment (denzalman) [inline home-assignments/session2/cli.py]: if you make such exception rising - make print statement with description of an error...

File: home-assignments/session2/cli.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        for index, forecast_day in enumerate(forecasts[slice(0, forecast + 1)]):
+            if index == 0:
+                print(f'The weather in {city} today is {forecast_day.text} with temperatures trailing from {forecast_day.low}-{forecast_day.high} {scale_name}.\n')
+            elif index == 1:
+                print(f'Forecast for the next {forecast} days:\n')
+                print(f'{forecast_day.date} {forecast_day.text} with temperatures trailing from {forecast_day.low}-{forecast_day.high} {scale_name}.')
+            else:
+                print(f'{forecast_day.date} {forecast_day.text} with temperatures trailing from {forecast_day.low}-{forecast_day.high} {scale_name}.')
+    except ValueError:
+        print('Number of the days for forecast should be less than 10')
```
Comment (cubann) [inline home-assignments/session2/cli.py]: Do mean this message?

Comment (cubann) [inline home-assignments/session2/cli.py]: @denzalman Do you mean this error message or another one?

Comment (denzalman) [reply home-assignments/session2/cli.py]: yes

File: home-assignments/session2/cli.py (hunk 1 above)
Comment (cubann) [reply home-assignments/session2/cli.py]: @denzalman and without if-else how can I assign proper scale_name?

**Pull request merged**